core/src/wallet/ethereum/api_impl/EthereumLikeOperation.h

```diff
@@ -47,6 +47,8 @@ namespace ledger {
         private:
             std::shared_ptr<api::EthereumLikeTransaction> _transaction;
             std::vector<std::shared_ptr<api::InternalTransaction>> _internalTxs;
+            std::shared_ptr<OperationApi> _backend;
+            bool _initialized;
```

Comment:
That’s a convoluted name. What’s that for?

Reply:
For lazy loading of internal transactions

Reply:
LGTM, even though I think that comment should be in the code.

Reply:
Agreed. Such a variable name could be clear at a time and completely obscur later

Reply:
I renamed it so it will be clearer !